frontend/src/features/business/api/fetchCustomer.ts

```diff
@@ -0,0 +1,28 @@
+import { supabase } from '@/lib/api-client';
+import { transformBackendCustomerToFrontend } from '@/types/customer';
```

Comment:
nit: will it be a pain to rename from customer to something like "profile"? since the brides aren't our customers